paasta_tools/marathon_serviceinit.py

```diff
@@ -98,19 +95,38 @@ def get_bouncing_status(service, instance, client, job_config):
         return PaastaColors.red("Unknown (count: %s)" % app_count)
 
 
+def get_bouncing_status(service, instance, client, job_config):
+    apps = marathon_tools.get_matching_appids(service, instance, client)
+    app_count = len(apps)
+    bounce_method = job_config.get_bounce_method()
+    return bouncing_status_human(app_count, bounce_method)
+
+
+def desired_state_human(desired_state, instances):
+    if desired_state == 'start' and instances != 0:
+        return PaastaColors.bold('Started')
+    elif desired_state == 'start' and instances == 0:
+        return PaastaColors.bold('Stopped')
+    elif desired_state == 'stop':
+        return PaastaColors.red('Stopped')
+    else:
+        return PaastaColors.red('Unknown (desired_state: %s)' % desired_state)
+
+
+def get_desired_state(desired_state, instances):
+    return desired_state_human(desired_state, instances)
+
+
 def status_desired_state(service, instance, client, job_config):
     status = get_bouncing_status(service, instance, client, job_config)
-    desired_state = job_config.get_desired_state_human()
+    desired_state = get_desired_state(job_config.get_desired_state(), job_config.get_instances())
     return "State:      %s - Desired state: %s" % (status, desired_state)
 
 
-def status_marathon_job(service, instance, app_id, normal_instance_count, client):
+def status_marathon_job_human(service, instance, deploy_status, app_id=None,
```

Comment:
please remove the defaults from here. Make it the caller's responsibility to provide sane values for these - this function should really just encode a template to print.